### PR TITLE
Give moderators more control over the stories

### PR DESCRIPTION
**Pull request by silent1mezzo**
* Allow moderators to Undelete and Unapprove stories
* Allow moderators the ability to show already approved stories

### CODE REVIEW

**Comment by silent1mezzo** [comment]
This closes #86

**Comment by silent1mezzo** [comment]
View at

File: pages/_admin/index.tsx [hunk 1]
```diff
@@ -100,12 +119,13 @@ export async function getServerSideProps({ req, query }) {
   const session = await getSession({ req })
 
   const deleted = query.deleted ? true : false
```
Comment (JamiesonRoberts) [inline pages/_admin/index.tsx:121]: Technically both of these could be simplified to be `!!query.deleted` and `!!query.approved` but I personally find that syntax super confusing when I read it so it probably makes sense to leave it as a ternary!

Comment (davesteinberg) [reply pages/_admin/index.tsx:121]: How about just `query.deleted === true`? Maybe the clearest and most direct option of all?
As general rule, ternaries with `true : false` can always be converted to a strict equals comparison to true or false.

**Review by mwickett — APPROVED** [review]
Looks good (I agree with the strict equals point from @davesteinberg)

Minor improvement suggestion: trigger a page reload after clicking a delete / approve / unapprove button so that the UI updates to reflect the change.

**Pull request merged**